test/ex_machina/ecto_test.exs

```diff
@@ -42,4 +42,22 @@ defmodule ExMachina.EctoTest do
       TestFactory.params_for(:user_map)
     end
   end
+
+  test "params_with_assocs/2 inserts belongs_tos that are set by the factory" do
+    assert Enum.member?(ExMachina.Article.__schema__(:associations), :editor)
```

Comment:
What do you mean by `I'm not exactly sure what "editor" looks like when it's not set`? If somebody adds `:editor` to the factory this test will fail because it will be included in the hash returned by `params_with_assocs`.